docs/asyncapi.yaml

```diff
@@ -1,13 +1,12 @@
 asyncapi: 2.2.0
```

Comment:
Should this be 2.6.0?

Reply:
Yeah it should.  I've updated it.  Probably could do a bit of general tidying up, but updating the spec to the latest version seems a good quick win